ddtrace/appsec/ddwaf/__init__.py

```diff
@@ -0,0 +1,155 @@
+import ctypes
+import logging
+import time
+from typing import TYPE_CHECKING
+
+from six import text_type
+
+
+if TYPE_CHECKING:
+    from typing import Any
+    from typing import Union
+
+    from ddtrace.appsec.ddwaf.ddwaf_types import DDWafRulesType
+
+
+LOGGER = logging.getLogger(__name__)  # type: logging.Logger
+
+try:
+    from .ddwaf_types import ddwaf_config
+    from .ddwaf_types import ddwaf_context_destroy
+    from .ddwaf_types import ddwaf_context_init
+    from .ddwaf_types import ddwaf_destroy
+    from .ddwaf_types import ddwaf_get_version
+    from .ddwaf_types import ddwaf_init
+    from .ddwaf_types import ddwaf_object
+    from .ddwaf_types import ddwaf_result_free
+    from .ddwaf_types import ddwaf_ruleset_info
+    from .ddwaf_types import ddwaf_update_rule_data
+    from .ddwaf_types import py_ddwaf_required_addresses
+    from .ddwaf_types import py_ddwaf_run
+
+    _DDWAF_LOADED = True
+except OSError:
+    _DDWAF_LOADED = False
+    LOGGER.warning("DDWaf features disabled. WARNING: Dynamic Library not loaded")
```

Comment:
```suggestion
    LOGGER.warning("DDWaf features disabled. WARNING: Dynamic Library not loaded", exc_info=True)
```

Would the `OSError` details be helpful for debugging?